examples/bri-3/src/bri/workgroup/workflows/api/workflows.controller.spec.ts

```diff
@@ -230,15 +240,11 @@ describe('WorkflowsController', () => {
 
     it('should perform the delete if existing id passed', async () => {
       // Arrange
-      const workflowId = await createTestWorkflow();
+      const workflow = createTestWorkflow();
+      workflowStorageAgentMock.getWorkflowById.mockResolvedValueOnce(workflow);
 
       // Act
-      await workflowController.deleteWorkflow(workflowId);
-
-      // Assert
-      expect(async () => {
-        await workflowController.getWorkflowById(workflowId);
-      }).rejects.toThrow(new NotFoundException(WORKFLOW_NOT_FOUND_ERR_MESSAGE));
+      await workflowController.deleteWorkflow(workflow.id);
```

Comment:
no way to assert it, pretty much by letting it execute it showcase there is no error thrown prisma, since this is just calling prisma delete function and nothing else